src/bbtag/subtags/guild/guildsize.ts

```diff
@@ -21,7 +21,8 @@ export class GuildSizeSubtag extends CompiledSubtag {
         });
     }
 
-    public getMemberCount(context: BBTagContext): number {
-        return context.guild.memberCount;
+    public async getMemberCount(context: BBTagContext): Promise<number> {
+        await context.util.ensureMemberCache(context.guild);
+        return context.guild.members.size;
```

Comment:
I had to change it because the `guild_member_count` property disappeared from the ApiGuild type in the discord api, so I couldnt set it up for the tests any more. This is more accurate anyway!